cli/deployments/quarks.go

```diff
@@ -49,12 +49,26 @@ func (k Quarks) Describe() string {
 func (k Quarks) Delete(c *kubernetes.Cluster, ui *ui.UI) error {
 	ui.Note().Msg("Removing Quarks...")
 
+	message := "Deleting Quarks namespace " + QuarksDeploymentID
+	warning, err := helpers.WaitForCommandCompletion(ui, message,
+		func() (string, error) {
+			return c.DeleteNamespaceIfOwned(QuarksDeploymentID)
+		},
+	)
+	if err != nil {
+		return errors.Wrapf(err, "Failed deleting namespace %s", QuarksDeploymentID)
+	}
+	if warning != "" {
+		ui.Exclamation().Msg(warning)
+		return nil
```

Comment:
Same as above

Reply:
Added a comment and exiting only for a specific warning.